src/main/webapp/dataset.xhtml

```diff
@@ -1525,9 +1525,39 @@
                     <!-- Publish/Submit for Review Dialogs -->
                     <p:dialog id="publishDataset" width="70%" header="#{bundle['dataset.publish.header']}" widgetVar="publishDataset" modal="true">
                         <ui:fragment  rendered="#{publishDataset}">
-                            <p class="text-warning">
-                                <span class="glyphicon glyphicon-warning-sign"/> #{bundle['dataset.publish.tip']}
-                            </p>
+                            <div class="form-group">
+                                <p class="col-sm-12 text-warning">#{bundle['dataset.publish.tip']}</p>
+                                <p class="col-sm-12 help-block">This version of the dataset will be published with the
```

Comment:
This text should also be configured in the bundle properties

Reply:
ah yes of course